stepper-lib/src/errors.rs

```diff
@@ -93,6 +91,34 @@ pub enum AquamarineError {
     ShadowingError(String),
 }
 
+impl AquamarineError {
```

Comment:
I'm so happy this mapping finally found its true place!

Reply:
And me too :)